packages/p2p/src/components/my-ads/create-ad-summary.jsx

```diff
@@ -7,17 +7,43 @@ import { buy_sell } from 'Constants/buy-sell';
 import { Localize } from 'Components/i18next';
 import { useStores } from 'Stores';
 
-const CreateAdSummary = ({ offer_amount, price_rate, type }) => {
+const CreateAdSummary = ({ market_feed, offer_amount, price_rate, type }) => {
     const { general_store } = useStores();
     const { currency, local_currency_config } = general_store.client;
 
     const display_offer_amount = offer_amount ? formatMoney(currency, offer_amount, true) : '';
-    const display_price_rate = price_rate ? formatMoney(local_currency_config.currency, price_rate, true) : '';
-    const display_total =
-        offer_amount && price_rate ? formatMoney(local_currency_config.currency, offer_amount * price_rate, true) : '';
+
+    const display_price_rate = (() => {
+        if (!!market_feed && !!price_rate) {
```

Comment:
```suggestion
        if (!!market_feed && !!price_rate) {
```
Is it necessary to coerce to boolean here?

Reply:
It works the same without coercing to boolean as well, since the input is in string